### ISSUE TITLE
Absolute path should be relative

### ISSUE THREAD

**Issue by wtworden** [3-manifolds/Sage_macOS#13]
There appears to be an instance of an absolute path (/Users/culler/programs/Sage_macOS/...) that should be a relative path.

For context, I am trying to get Regina working inside Sage_macOS. I installed it via %pip install sageRegina (well, actually it was installed as a dependency via %pip install tnorm). After installing, importing Regina raises an ImportError because Regina is trying to use the version of Python bundled with the Regina app, which is 3.8:

`ImportError: Python version mismatch: module was compiled for Python 3.8, but the interpreter version is incompatible: 3.9.2 (default, Jun 10 2021, 21:07:27)
[Clang 12.0.5 (clang-1205.0.22.9)].`

A solution to this is to force regina to ignore this Python instance by doing 

`sage: sys.path.remove('/Applications/Regina.app/Contents/MacOS/python')`

After doing this, importing Regina raises another error, which is the one that this issue is referring to. This seems to be caused by an attempt to load something from an absolute path which only exists on Marc's computer:

`ImportError: dlopen(/var/tmp/sage-9.3-current/local/lib/python3.9/site-packages/regina/engine.cpython-39-darwin.so, 2): Library not loaded: /Users/culler/programs/Sage_macOS/Sage_framework/repo/sage/local/lib/libgmp.10.dylib
  Referenced from: /var/tmp/sage-9.3-current/local/lib/python3.9/site-packages/regina/engine.cpython-39-darwin.so
  Reason: image not found`

Thanks!!

**Comment by culler** [comment]
Thank you, Will.  I believe that there are some config.log files containing
that absolute path (left over from when I ran configure in the sage repo),
and that the path has found its way from the config.log file into
distutils.  We also saw this when building SnapPy.  We are investigating
how to fix it.

It would not be hard to directly fix the path in Regina's engine library.
But it would be better if Regina never encountered the absolute path in the
first place.

- Marc
